meinberlin/apps/projects/models.py

```diff
@@ -71,3 +72,70 @@ def get_absolute_url(self):
     def accept(self, user):
         self.project.moderators.add(user)
         super().accept(user)
+
+
+class ProjectInsight(base.TimeStampedModel):
+    project = models.OneToOneField(
+        Project, related_name="insight", on_delete=models.CASCADE
+    )
+    active_participants = models.ManyToManyField(settings.AUTH_USER_MODEL)
+    unregistered_participants = models.PositiveIntegerField(default=0)
+    comments = models.PositiveIntegerField(default=0)
+    ratings = models.PositiveIntegerField(default=0)
+    written_ideas = models.PositiveIntegerField(default=0)
+    poll_answers = models.PositiveIntegerField(default=0)
+    live_questions = models.PositiveIntegerField(default=0)
+
+    @staticmethod
+    def update_context(project, context):
+        insight, created = ProjectInsight.objects.get_or_create(project=project)
+        context.update(create_insight_context(insight=insight))
+        return context
+
+    def __str__(self):
+        return "Insights for project %s" % self.project.name
```

Comment:
few things here ( I know you imported them from aplus, but always room for improvement):
Change the staticmethod to classmethod, then we can pass `cls` in the arguments of the functions, instead of calling `ProjectInsight` which is odd since we are inside the class already.
`

Use f-strings in `__str__()`, more readable

We may want to add the `Meta` attribute to define the ordering of the objects `(-created)`